tests/Integration/IntegrationTest.php

```diff
@@ -0,0 +1,206 @@
+<?php
+
+declare(strict_types=1);
+
+namespace T3Docs\Typo3DocsTheme\Integration;
+
+use function array_filter;
+use function array_merge;
+use function array_walk;
+use function assert;
+use function escapeshellarg;
+use function explode;
+use function file_exists;
+use function file_get_contents;
+use function implode;
+
+use const LC_ALL;
+
+use phpDocumentor\Guides\Cli\Command\Run;
+use PHPUnit\Framework\Attributes\DataProvider;
+use PHPUnit\Framework\ExpectationFailedException;
+
+use function setlocale;
```

Comment:
Do we really need those "use function" thingy? It clutters the "use" section IMHO. Either avoid it or add a `\` before the functions when using it to signal it is in the global namespace.

Reply:
I believe this is the doctrine code-style. But honestly I wouldnt want to change these files where not absolutely nessesary as it will make it harder to cherry pick changes from https://github.com/phpDocumentor/guides/blob/main/tests/ApplicationTestCase.php and https://github.com/phpDocumentor/guides/blob/main/tests/Integration/IntegrationTest.php into this test. I would suggest that you make a PR to change those things that make sense in the original files instead?

Reply:
Yes, can be done in a separate PR. IMHO it should be a setting in php-cs-fixer.